### DISCUSSION TITLE
CF 部署教程有点晦涩难懂，有么有详细的流程？

### DISCUSSION THREAD

**Discussion by cyclestudy** [dreamhunter2333/cloudflare_temp_email#472] [General]

**Comment by dreamhunter2333** [comment]
目前只有这份文档

https://temp-mail-docs.awsl.uk/

**Comment by lavelois** [comment]
其实有点多线程😂，可以参考我的过程试试：

[mail.pdf](https://github.com/user-attachments/files/17574525/mail.pdf)
